main.go

```diff
@@ -31,6 +32,12 @@ import (
 	"github.com/ziflex/lecho/v3"
 )
 
+//go:embed templates/index.html
+var html string
```

Comment:
maybe indexHtml? to be more descriptive that this is specifically the index page?